src/components/LayoutManager/LayoutManager.tsx

```diff
@@ -94,7 +95,8 @@ export const LayoutManager: React.FC<LayoutManagerProps> = ({
         }
         setIsSidebarCollapsed((state) => {
           const isOpen = !state;
-          localStorage &&
+          canAccessLocalStorage() &&
+            localStorage &&
```

Comment:
on these checks, do we need the `&& localStorage` here if we've already established `canAccessLocalStorage()`?

Reply:
you're right,  `canAccessLocalStorage` will catch that 👍